tests/Functional/Net/SFTPUserStoryTest.php

```diff
@@ -649,5 +649,22 @@ public function testStatVsLstat($sftp)
         $this->assertSame($stat['type'], NET_SFTP_TYPE_SYMLINK);
 
         $sftp->enableStatCache();
+
+        return $sftp;
+    }
+
+    /**
+     * @depends testStatVsLstat
+     * @group github830
+     */
+    public function testEndlessLoopOnUpload($sftp)
```

Comment:
SFTP.php is still doing the fseek(). Or at least was. But regardless, the latest commit removes that. If I'm going to have a unit test for that portion of SFTP.php then that portion of SFTP.php ought to be behaving as per the documentation.